symplyphysics/laws/thermodynamics/isobaric_reaction_potential.py

```diff
@@ -0,0 +1,50 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The isobaric potential of a reaction is a value whose change during a chemical reaction is equal to the change in the internal
+## energy of the system. The isobaric potential shows how much of the total internal energy of the system can be used for chemical
+## transformations.
+
+## Law is: G = H - T * S, where
+## G - isobaric potential of reaction,
```

Comment:
Why isobaric?

Reply:
Because it describes an isobaric process.

Reply:
Of course. But it probably should be in conditions.